### PR TITLE
feat: add more credentials

### LINKED ISSUE #2: Add additional client credentials
Currently only using one set of credentials

### PR DESCRIPTION
**Pull request by nickchen120235**
Closes #2

This PR introduces all four key pairs found [here](https://git.sr.ht/~cloutier/bird.makeup/tree/master/item/src/BirdsiteLive.Twitter/Tools/TwitterAuthenticationInitializer.cs#L36)

## Testing Instructions
No functionality changes. Everything should still work.

### CODE REVIEW

File: src/twitter.ts [hunk 1]
```diff
@@ -8,12 +8,18 @@ const userIdAPIForUsername = (username: string) => `https://api.twitter.com/grap
...
+  ['CjulERsDeqhhjSme66ECg', 'IQWdVyqFxghAtURHGeGiWAsmCAGmdW3WmbEx6Hck', 'iPad'],
+  // ['3rJOl1ODzm9yZy63FACdg', '5jPoQ5kQvMJFDYRNE8bQ4rHuds4xJqhvgNJM4awaE8', 'Mac'],
+];
 
 const fetchAccessToken = async (): Promise<string> => {
-  const [credentials] = CLIENT_CREDENTIALS;
+  // pick one randomly
+  const [username, password, client] = CLIENT_CREDENTIALS[Math.floor(Math.random() * CLIENT_CREDENTIALS.length)];
+  const credentials = Buffer.from(`${username}:${password}`).toString('base64');
+  console.log(`Using crednetial: ${client}`)
```
Comment (12joan) [inline src/twitter.ts:22]:
```suggestion
  console.log(`Using credentials: ${client}`)
```

**Review by 12joan — CHANGES_REQUESTED** [review]
Looks good! Just one small typo

**Comment by nickchen120235** [comment]
I'm converting this to draft temporarily because I found some other tokens that might work. https://github.com/zedeus/nitter/pull/925

I'll be back if any (or none) of them works.

**Comment by nickchen120235** [comment]
So both the `developer.twitter.com` and the `ipad` token will return 20 tweets per request. But the `tweetdeck` token usually returns less than 20 tweets.

**Comment by 12joan** [comment]
That's interesting. I'll let you decide what's best here.

Incidentally, I'm liking Twitter's new tagline, courtesy of DuckDuckGo.
Search result for TweetDeck. The text under the result is 'Something went wrong, but don't fret — let's give it another shot.'

**Comment by nickchen120235** [comment]
So the options are
1. Keep the `tweetdeck` key, pick the keys randomly
2. 1. but give the `tweetdeck` key a lower weight (I'm thinking 20% for it and 40% for each of the other two keys)
3. 1. but we DO NOT cache the `tweetdeck` key (actually we may not need to cache the bearer key anymore because they are hard-coded)
4. Remove the `tweetdeck` key

**Comment by 12joan** [comment]
That sounds all right to me. No need to remove guest token caching, since [the cache is keyed on the access token](https://github.com/12joan/twitter-client/blob/main/src/twitter.ts#L141).

**Comment by 12joan** [comment]
Are we confident that the hardcoded access tokens are just as unlikely to change as the raw client credentials?

**Comment by nickchen120235** [comment]
As far as I can tell, the bearer token for the frontend (`twitter.com`) hasn't changed untill all hell breaks loose, so I think it's unlikely to change.

However, considering the Tesla guy is constantly unplugging any means of twitter profile scraping, the tokens may or may not be invalid in the near future.

File: src/twitter.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ const userIdAPIForUsername = (username: string) => `https://api.twitter.com/grap
...
 
-const fetchAccessToken = async (): Promise<string> => {
+const fetchAccessToken = (): string => {
```
Comment (12joan) [inline src/twitter.ts:10]: Looks good! Might as well rename this to `get[Random]AccessToken` or something like that now that we're not fetching anything.

**Review by 12joan — APPROVED** [review]

**Comment by 12joan** [comment]
Thanks! Are we ready to merge?

**Comment by nickchen120235** [comment]
I think we are good to go!

**Pull request merged**